lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -304,6 +327,93 @@ impl Client {
         })
     }
 
+    async fn handle_clipboard(
+        cgo_handle: CgoHandle,
+        write_stream: &mut RdpWriteStream,
+        x224_processor: Arc<Mutex<X224Processor>>,
+        clipboard: &mut Option<String>,
+        clipboard_function: ClipboardFunction,
+    ) -> ClientResult<()> {
+        match clipboard_function {
+            ClipboardFunction::RequestFormatList => {
+                Client::process_cliprdr(x224_processor, write_stream, |c| c.initiate_copy(&[]))
+                    .await?
+            }
+            ClipboardFunction::Update(data) => {
+                clipboard.replace(data);
+                Client::process_cliprdr(x224_processor, write_stream, |c| {
+                    c.initiate_copy(&[ClipboardFormat::new(ClipboardFormatId::CF_UNICODETEXT)])
```

Comment:
Noting that we've lost this ascii vs unicode capability here, what's your reasoning?

https://github.com/gravitational/teleport/blob/e19bfee84116c17390847a92f65dae92c6afbfb1/lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs#L552-L567

Reply:
Windows should handle unicode in almost all situation and the code was much simpler. I did add back `CF_TEXT` support, though, for feature parity